_drafts/bridging-digital-and-physical.markdown

```diff
@@ -0,0 +1,140 @@
+---
+layout:       post
+title:        Bridging digital and physical with a MakerBot Replicator 2
+author:       Dave Mills
+summary:
+image:        http://image.shutterstock.com/display_pic_with_logo/1451378/316560056/stock-vector-pumpkin-dressed-up-in-stripy-suit-furry-art-character-halloween-greeting-design-316560056.jpg
```

Comment:
I think it's fine to leave the other images stored in the assets folder, since we own that. @i8ramin -- any reason they should be uploaded directly to Cloudinary instead of storing them within the app assets?